.github/workflows/dep_auto_merge.yml

```diff
@@ -12,6 +12,8 @@ jobs:
   dependabot-auto-merge:
     name: Merge a dependabot PR
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
```

Comment:
I guess this doesn't work. Have you tested this before?

I remember only the `pull_request_target` can have the write permission. But obviously, it's dangerous.

Reply:
Haven't test yet, but there are some comments at plugin we used about this:
ref:
https://github.com/mshick/add-pr-comment/issues/46
https://github.com/mshick/add-pr-comment/pull/48
https://github.com/anothrNick/github-tag-action/issues/154

Reply:
I don't think so. You can try.

Reply:
Then should we merge this to try?

Reply:
You can merge this. But if it doesn't work, I don't think we should waste time on GitHub Actions.

Reply:
> Then should we merge this to try?

Oh, I could try on a fork

Reply:
Yep, the manual merge also LGTM

Reply:
It solved, but there seems to be another permission problem about this.
https://github.com/cutecs/envd-lsp/pull/1